wordpress-mobile/WordPress-Android
# App Data Coverage: Add more tracking for people management

Parent #16343 

This PR adds more tracking for People management

~~**Merging Notes**~~
~~Please do not merge, just approve. I have another PR [#16429] that will conflict with events added to tracking enums and depending on the order of merge, I may need to resolve conflicts. Thanks.~~

**To test:**
- Launch the app and log in
- Navigate to My Site “menu” tab
- Tap the “people” menu item
- Tap the dropdown filter and change the value
- ✅ Verify the logs contain: `🔵 Tracked: people_management_filter_changed
- Tap the “+” menu item to launch the invite flow
- Enter the invite flow details and tap SEND
- ✅ Verify the logs contain: `🔵 Tracked: people_management_user_invited`

## Regression Notes
1. Potential unintended areas of impact
None, add new tracks

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
